### PR TITLE
fixed #9 use `ava` instead of `npm t` for `cover`

### LINKED ISSUE #9: npm run validate errors out at npm run check-coverage
Am I the only one finding error for `npm run validate`, to be precise `npm run check-coverage`

`coverage` directory does exist, I have verified it with `npm run cover`.

```
ERROR: check-coverage: None-Zero Exit(1);

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "validate"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! eslint-find-new-rules@0.0.0-semantically-released validate: `npm-run-all --parallel lint cover --sequential check-coverage`
npm ERR! Exit status 1
```

---

```
吽 npm run check-coverage

> eslint-find-new-rules@0.0.0-semantically-released check-coverage /Users/sabandyo/GitHub/eslint-find-new-rules
> nyc check-coverage --statements 71 --branches 33 --functions 100 --lines 71

ERROR: No coverage files found.

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "check-coverage"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! eslint-find-new-rules@0.0.0-semantically-released check-coverage: `nyc check-coverage --statements 71 --branches 33 --functions 100 --lines 71`
npm ERR! Exit status 1
```

Thanks!!

### PR DESCRIPTION
**Pull request by sarbbottam**
The `ERROR: No coverage files found.` issue gets resolved by passing the `--coverage` option to the `npm run cover`

screen_shot_2016-03-17_at_11_10_18_am

### CODE REVIEW

**Comment by sarbbottam** [comment]
@kentcdodds, @ta2edchimp could you take a look? Thanks!

File: package.json [hunk 1]
```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
```
Comment (sarbbottam) [inline package.json:7]: I generally always run the test with coverage.

```
"lint": "eslint ./src/* ./test/*",
"unit-test": "nyc --reporter=lcov mocha --compilers js:babel-core/register",
"coverage": "nyc check-coverage --statement 100 --branches 100 --function 100 --lines 100",
"test": "npm run lint && npm run unit-test --coverage && npm run coverage",
```

File: package.json [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```
Comment (kentcdodds) [inline package.json]: That's odd... `--coverage` doesn't appear to be a flag that `ava` responds to at all... But you say this fixes the problem for you?

Comment (sarbbottam) [inline package.json]: Yea, you could find the result in the screenshot attached in PR :smile:

Comment (kentcdodds) [inline package.json]: Hmm... And you're sure it wasn't working before? Could you try removing the `--coverage` from this line and showing me the output for that? The reason I'm so skeptical is because that's not a documented flag for `ava` and honestly doesn't really make sense to me.

Comment (ta2edchimp) [inline package.json]: I fear, it's not only not documented, but also not even mentioned [in the sources](https://github.com/sindresorhus/ava/blob/master/cli.js#L57-L107) anywhere ...

Comment (sarbbottam) [inline package.json]: Please have a look.

nyc-coverage

Comment (ta2edchimp) [inline package.json]: As I observed now (at least in my environment): the `cover` script outputs the `.nyc_output` and `coverage` folders even with the original `npm t` command; that is, `ava` without `--coverage`.
The latter does not change anything. But that would basically be, as it is without this pr's changes.

Comment (ta2edchimp) [inline package.json]: Could you please check your `node` and `npm` versions and report them back here? Like

```
$ node --version
v5.8.0
$ npm --version
3.8.1
```

There seems to be an issue with `nyc` running on node `5.8.0` and npm `3.7.3`, as reported [here](https://github.com/bcoe/nyc/issues/190).

Comment (kentcdodds) [inline package.json]: Hmmm... @sarbbottam could you try with:

```
"cover": "nyc --reporter=lcov --reporter=text ava"
```

Does that make a difference?

Comment (kentcdodds) [inline package.json]: You might also try: 

```
"cover": "nyc --reporter=lcov --reporter=text npm t --coverage"
```

Just to try more stuff out.

Thanks for your work on this :+1:

Comment (sarbbottam) [inline package.json]: That has been the first option I tried, does not work. :cry: 

nyc-coverage-1

Comment (kentcdodds) [inline package.json]: Have you tried:

```
"cover": "nyc --reporter=lcov --reporter=text ava"
```

?

Comment (sarbbottam) [inline package.json]: Similar fix for https://github.com/kentcdodds/configuration-validator/issues/9 
Please refer https://github.com/kentcdodds/configuration-validator/issues/9#issuecomment-198073323

Comment (sarbbottam) [inline package.json]: > > Have you tried: `"cover": "nyc --reporter=lcov --reporter=text ava"`?

Thats a good catch!! It works, So nothing to do with passing `--coverage` but **not use `npm test`**

Let me update the PR.

**Comment by kentcdodds** [comment]
Cool. This looks good. I think that `nyc` people may want to know about this.

TL;DR - Using `npm t` doesn't work on some systems (apparently). Maybe someone should look into that...

**Comment by kentcdodds** [comment]
Would you like to add yourself to the contributor's list? https://github.com/kentcdodds/eslint-find-new-rules/blob/master/.all-contributorsrc
